waldo/app/routers/EndpointRouter.js

```diff
@@ -4,14 +4,10 @@ const cors = require ('cors');
 const gatekeeper = require('@onehilltech/blueprint-gatekeeper');
 
 const corsOptions = {
-  methods: ['GET', 'POST', 'PUT', 'PATCH', 'DELETE']
+  methods: ['GET', 'POST', 'PUT', 'PATCH', 'DELETE'],
+  origin: "*"
 };
 
 module.exports = {
-  '/v1': {
```

Comment:
This seems counter intuitive, but it still protects the route and it lets cors through.